stub-environment/environments/Test-Laptop.json

```diff
@@ -72,7 +76,7 @@
       "config": {
          "verify_api_cert": false,
          "ssl_verify_mode": "verify_none",
-         "rubygems_url": "http://10.0.100.3/"
+         "rubygems_url": "http://10.0.101.3/"
```

Comment:
Your cluster.txt stub has 10.0.100 but you're setting 101 here...

Reply:
It hardly matters what we have in cluster.txt as we override it anyway (?)

Reply:
We do not override the IP addresses that I can think of?